### PR TITLE
Add batch post view

### PR DESCRIPTION
**Pull request by teodorus-nathaniel**

### CODE REVIEW

File: src/components/posts/PostViewSubmitter.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  useEffect(() => {
+    if (!myAddress) return
+
+    const intervalId = setInterval(async () => {
+      try {
+        const postViewsString = postViewStorage.get()
+        if (!postViewsString) return
+        const postViews = JSON.parse(postViewsString) as string[]
+        if (!Array.from(postViews)) return
```
Comment (olehmell) [inline src/components/posts/PostViewSubmitter.tsx:20]: why not isArray check?

File: src/components/posts/PostViewSubmitter.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+        const filteredIds = new Set<string>()
+        postViews.forEach(id => {
+          if (typeof id !== 'string') return
+          filteredIds.add(id)
+        })
```
Comment (olehmell) [inline src/components/posts/PostViewSubmitter.tsx:26]: Is it possible to be not string?

File: src/components/posts/view-post/PostPreview.tsx [hunk 3]
```diff
@@ -66,26 +66,19 @@ export function PostPreview(props: PreviewProps) {
...
-              args: {
-                viewerId: myAddress,
-                duration: POST_VIEW_DURATION,
-                postPersistentId: originalPostId,
-              },
-            }),
-          )
+        let views = JSON.parse(postViewStorage.get() || '[]') as string[]
+        if (!Array.isArray(views)) {
+          views = []
```
Comment (olehmell) [inline src/components/posts/view-post/PostPreview.tsx:71]: copy-paste with PostPage code, do some wrapper

**Review by olehmell — CHANGES_REQUESTED** [review]

File: src/components/posts/PostViewSubmitter.tsx (hunk 2 above)
Comment (teodorus-nathaniel) [reply src/components/posts/PostViewSubmitter.tsx:26]: should not, but if the user just changes something, I just make a vlaidation for it

**Pull request merged**